tests/regression/nextcloud-rule-exclusions-plugin/9508170.yaml

```
@@ -0,0 +1,40 @@
+---
+meta:
+  author: "Esad Cetiner"
+  description: "Nextcloud Rule Exclusions Plugin"
+  enabled: true
+  name: 9508170.yaml
+tests:
+  - test_title: 9508170-1
+    desc: Viewing files stored in the trashbin
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Host: localhost
+              User-Agent: OWASP CRS test agent
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+              Content-Type: text/plain;charset=UTF-8
+            port: 80
+            method: PROPFIND
+            uri: /remote.php/dav/trashbin/username/trash/
+            data: >-
+              <?xml version="1.0"?>
+              <d:propfind xmlns:d="DAV:" xmlns:nc="http://nextcloud.org/ns" xmlns:oc="http://owncloud.org/ns" xmlns:ocs="http://open-collaboration-services.org/ns">
+                      <d:prop>
+                              <nc:trashbin-deletion-time />
+                              <nc:trashbin-original-location />
+                              <nc:trashbin-title />
+                              <nc:trashbin-deleted-by-id />
+                              <nc:trashbin-deleted-by-display-name />
+                              <d:getcontentlength /> <d:getcontenttype /> <d:getetag /> <d:getlastmodified /> <d:creationdate /> <d:displayname /> <d:quota-available-bytes />
+                               <d:resourcetype /> <nc:has-preview /> <nc:is-encrypted /> <nc:mount-type /> <oc:comments-unread /> <oc:favorite /> <oc:fileid /> <oc:owner-display-name /> <oc:owner-id />
+                               <oc:permissions /> <oc:size /> <nc:hidden /> <nc:is-mount-root /> <nc:metadata-files-live-photo /> <nc:reminder-due-date /> <nc:note /> <nc:sharees />
+                               <nc:share-attributes /> <oc:share-types /> <ocs:share-permissions />
+                       </d:prop>
+               </d:propfind>
+            version: HTTP/1.1
+          output:
+            no_log_contains: |
```

Careful with block scalars. The trailing newline would change your regular expression to `...|id "921110"\n`.
```suggestion
            no_log_contains: |-
```